Doc/library/crypt.rst

```diff
@@ -20,6 +20,14 @@
    The :mod:`crypt` module is deprecated
    (see :pep:`PEP 594 <594#crypt>` for details and alternatives).
    The :mod:`hashlib` module is a potential replacement for certain use cases.
+   The :mod:`passlib` package can replace all use cases of this module.
+
+    >>> import crypt
+    >>> crypt.crypt("cleartext", salt="ab")
+    'ab/GpyA5I.S12'
+    >>> from passlib.hash import des_crypt
+    >>> des_crypt.hash("cleartext", salt="ab")
```

Comment:
Something like this should skip the doctest failure:

```suggestion
    >>> from passlib.hash import des_crypt  # doctest: +SKIP
    >>> des_crypt.hash("cleartext", salt="ab")  # doctest: +SKIP
```

Reply:
Fixed, thanks